.gitignore

```diff
@@ -12,3 +12,4 @@ venv/
 .Rhistory
 .Rproj.user*
 .Rproj.user
+codelists/primis-covid19-vacc-uptake-ast.csv
```

Comment:
Please remove this codelist from the .gitignore otherwise we will not be able to use it in the code.